mscore/timeline.h

```diff
@@ -192,6 +192,8 @@ class Timeline : public QGraphicsView {
       Timeline(TDockWidget* dock_widget, QWidget* parent = 0);
       int correctPart(int stave);
 
+      QList<Part*> getParts();
```

Comment:
Better make it private perhaps? Or is it intended to be used outside `Timeline`?

Reply:
no, you're right, it should be private.